app/services/preserver/blind_importer.rb

```diff
@@ -33,6 +33,14 @@ def import!
     end
     # Get rid of non-preserved members.
     source_change_set.try(:member_ids=, member_ids)
+    # Delete any tombstones
+    source_change_set.created_file_sets.each do |_created_file_set|
```

Comment:
Why do you iterate over created file sets? It seems like you don't use the result.